web/docs/setup/tune.md

```diff
@@ -7,15 +7,13 @@ sidebar_position: 5
 This section describes tuning knobs that have a notable effect on system
 performance.
 
-## Batching: Table Slices
+## Batching
 
 VAST processes events in batches. Because the structured data has the shape of a
 table, we call these batches *table slices*. The following options control their
 shape and behavior.
 
-:::note Implementation Note
-Table slices are implemented as *Record Batches* in Apache Arrow.
-:::
+Internally, VAST implements a table slice as an Arrow *record batch*.
```

Comment:
I think this is a meaningless implementation detail for tuning. Let's just remove this sentence.